apps/admin/src/styles/tokens/index.css

```
@@ -0,0 +1,16 @@
+
+@import "./_admin-base.css";
+@import "./_admin-colors.css";
+@import "./_admin-semantic.css";
+@import "./_admin-spacing.css";
+@import "./_admin-typography.css";
+@import "./_admin-layout.css";
+@import "./_admin-icons.css";
+
+@import "./components/components.css";
+
+@plugin "tailwindcss-animate";
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)